test/cpp/end2end/end2end_test.cc

```diff
@@ -1206,9 +1206,9 @@ TEST_P(End2endTest, CancelRpcAfterStart) {
       s = stub_->Echo(&context, request, &response);
     });
     if (!GetParam().callback_server()) {
-      service_.ClientWaitUntilRpcStarted();
+      EXPECT_EQ(service_.WaitForRpcsWaitingForServerToContinue(1), 1);
```

Comment:
I think these call sites will need to be changed, now that you've renamed the methods.

Reply:
whoops! fixed